sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuAggregateExec.scala

```diff
@@ -44,7 +44,7 @@ import org.apache.spark.sql.catalyst.trees.TreeNodeTag
 import org.apache.spark.sql.catalyst.util.truncatedString
 import org.apache.spark.sql.execution.{ExplainUtils, SortExec, SparkPlan}
 import org.apache.spark.sql.execution.aggregate.{BaseAggregateExec, HashAggregateExec, ObjectHashAggregateExec, SortAggregateExec}
-import org.apache.spark.sql.rapids.{CpuToGpuAggregateBufferConverter, CudfAggregate, GpuAggregateExpression, GpuToCpuAggregateBufferConverter}
+import org.apache.spark.sql.rapids.aggregate.{CpuToGpuAggregateBufferConverter, CudfAggregate, GpuAggregateExpression, GpuToCpuAggregateBufferConverter}
```

Comment:
so if we are going to put things under org.apache.spark.sql.rapids.aggregate, should we also put the rest under com.nvidia.spark.rapids.aggregate? It could be done later, I am mostly curious.

Reply:
I only see the rest here is `GpuAggregateExec.scala` thus I didn't do it yet. Did I miss anything?

Reply:
`GpuBloomFilterAggregate` is under `com.nvidia.spark.rapids`. I think that's the only one

Reply:
That one is also similar to other aggregate function, not part of the exec code. So I'm going to put it in `org.apache.spark.sql.rapids.aggregate`.